CHANGELOG.md

```diff
@@ -5,4 +5,4 @@ master docs][master_docs].
 
 [`changelog::unreleased`]: src/changelog/unreleased.md
 [docs_rs]: https://docs.rs/winit/latest/winit/changelog/index.html
-[master_docs]: https://rust-windowing.github.io/winit/winit/changelog/index.html
+[master_docs]: https://rust-windowing.github.io/winit/winit/changelog/index.html
```

Comment:
There must be a new line in the end of file. Don't drop it.

Reply:
Thanks, done in 2f313c1ec433390e3f97868c149b882744e0d2cd.